hedera-node/cli-clients/src/main/java/com/hedera/services/cli/signedstate/DumpContractBytecodesSubcommand.java

```diff
@@ -215,15 +217,11 @@ void appendFormattedContractLines(@NonNull final StringBuilder sb, @NonNull fina
     void appendFormattedContractLine(@NonNull final StringBuilder sb, @NonNull final Contract contract) {
         sb.append(hexer.formatHex(contract.bytecode()));
 
-        final var ids = contract.ids();
-        if (withIds == DumpStateCommand.WithIds.YES && !ids.isEmpty()) {
-            // Output canonical id - we choose the minimum id (so it is deterministic)
-            final var sortedIds = ids.stream().sorted().toList();
+        if (withIds == DumpStateCommand.WithIds.YES && !contract.ids().isEmpty()) {
```

Comment:
Q: Would the `ids` assignment have been useful to make the call once?
What about?
```suggestion
        final var ids = contract.ids();
        if (withIds == DumpStateCommand.WithIds.YES && !ids.isEmpty()) {
```

Then further down
```
sb.append(ids.stream()...)
```

Reply:
`ids()` is a simple getter from a record; per Java convention it doesn't have the `get` prefix that signifies a getter, but that's what it is.

Reply:
(resolving this conversatino for merge, hope you agree Nana)